### ISSUE TITLE
Could NOT find Eigen3 (missing: Eigen3_DIR)

### ISSUE THREAD

**Issue by nyckmaia** [BlueBrain/HighFive#292]
I'm trying run CMake with USE_EIGEN option enable, but I got this error message:

```
Could NOT find Eigen3 (missing: Eigen3_DIR)
Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE) 
```
Please, look this image below:
![image](https://user-images.githubusercontent.com/4555404/74934433-f8bc3200-53c4-11ea-89cf-5ce12d6a969f.png)

Here is my Eigen folder:
![image](https://user-images.githubusercontent.com/4555404/74934668-7ed87880-53c5-11ea-8081-44200c0e3927.png)

**Could you help me?**

**My setup:**
 * Windows 10 x64
 * CMake 3.14.4
 * Eigen 3.3.7
 * HighFive 2.1.1
 * HDF5 1.10.6

**Comment by alkino** [comment]
You have to install eigen3 and give the install path to the `Eigen3_DIR` arg.

In `eigen3`:
`cmake .. && make -j install DESTDIR=FOO`

In `HighFive`:
`cmake -D Eigen3_DIR=<foobar>/FOO`

**Comment by ferdonline** [comment]
@nyckmaia I believe I have seen a similar error but it might be addressed in #304. Could you please test again with the latest master?

**Comment by nyckmaia** [comment]
@alkino Could I set the HighFive `EIGEN_DIR` without install Eigen?
I mean, Can I set the `EIGEN_DIR` using just the downloaded Eigen folder?

Today, I'm using Eigen in my C++ project just coping the header files to inside my project and works great.

So I **don't** have this path in my machine: `C:\Program Files (x86)\Eigen\`

Well, I tried to install Eigen just to enable the HighFive `USE_EIGEN` option, but the build failed.
Here is my VS 2019 console output:
```
C1060	compiler is out of heap space
```

So, is there another way to set the CMake HighFive `EIGEN_DIR` **without compile and install Eigen?**
